lib/googleauth/credentials_loader.rb

```diff
@@ -50,11 +50,10 @@ module CredentialsLoader
                             "s.googleusercontent.com".freeze
 
       CLOUD_SDK_CREDENTIALS_WARNING =
-        "Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most " \
-        "server applications use service accounts instead. If your application continues to use end user credentials " \
-        'from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about ' \
-        "service accounts, see https://cloud.google.com/docs/authentication/. To suppress this message, set the " \
-        "GOOGLE_AUTH_SUPPRESS_CREDENTIALS_WARNINGS environment variable.".freeze
+        "You are authenticating using user credentials." \
+        "For production, we recommend using service account credentials." \
+        "To learn more about service account credentials, see" \
+        "http://cloud.google.com/docs/authentication/external/set-up-adc-on-cloud".freeze
```

Comment:
Is the intent to drop the note about being able to suppress the message with an environment variable?

Reply:
I don't think so. I'll remove it. Thanks.